src/main/java/org/craftercms/core/store/impl/filesystem/FileSystemContent.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2007-2019 Crafter Software Corporation. All Rights Reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.craftercms.core.store.impl.filesystem;
+
+import org.craftercms.core.service.Content;
+
+import java.io.*;
```

Comment:
Why not `nio`?

Reply:
We can change it to `nio`, I just didn't want to change most of the io code that was already there.